COVIDWatch.xcodeproj/project.pbxproj

```diff
@@ -944,11 +944,11 @@
 			baseConfigurationReference = A236DDAE6CBC8116CEEE63D5 /* Pods-COVIDWatch iOS.debug.xcconfig */;
```

Comment:
Please revert all changes in this file so automation doesn't break

Reply:
pushed